entity-framework/core/providers/ibm/index.md

```diff
@@ -22,15 +19,15 @@ This database provider allows Entity Framework Core to be used with IBM Data Ser
 
 To work with IBM Data Server on Windows, install the [IBM.EntityFrameworkCore NuGet package](https://www.nuget.org/packages/IBM.EntityFrameworkCore).
 
-````text
+``` console
```

Comment:
I don't know. Which are the two? 

FWIW, I like using console because if you include $ or PM> it colorizes the prompt symbol, e.g.:

![image](https://user-images.githubusercontent.com/1253414/29105231-5a20f64e-7c80-11e7-90de-0587992c4172.png)

Which helps differentiate an PMC snippet from a regular command prompt. 

But on the other hand I don't like it because if you click on <kbd>Copy</kbd> you get the whole thing :disappointed::

> PM>  Install-Package IBM.EntityFrameworkCore

So I was considering getting rid of it. If I do that, I don't know that `console` buy us anything compared to e.g. `terminal` which is what the ASP.NET Core docs is using.